pkg/engine/armvariables.go

```diff
@@ -27,12 +26,6 @@ func GetKubernetesVariables(cs *api.ContainerService) map[string]interface{} {
 			k8sVars[k] = v
 		}
 
-		if common.IsNvidiaEnabledSKU(profile.VMSize) {
-			if cs.Properties.IsNVIDIADevicePluginEnabled() {
-				k8sVars["registerWithGpuTaints"] = "nvidia.com/gpu=true:NoSchedule"
```

Comment:
this might trigger a UT failure heads up

Reply:
passed locally for me